client/init.js

```diff
@@ -18,25 +22,46 @@ addEventListener('load', function() {
 
 		form.querySelector('button').disabled = false
 	}
-
+	/**
+	 *
+	 * @param  {string} name The name of the new user, the input. Should probably
+	 *                       validate this.
+	 * @return {stuff}      stuff, not quite sure what stuff is yet.
```

Comment:
I'm fine with leaving stuff, after I traced it and thought about it, Stuff actually fits. It's just stuff.